Src/DataStreamManager.cs

```diff
@@ -941,12 +956,17 @@ public double GetPMData(string label)
         }
 
         /// <summary>
-        /// Start refresh (scan) headset automatically if isAutoRefreshHeadset = true. 
-        /// It mean when receive warning inform the scanning finish (142), It will call refresh headset again.
-        /// The refreshing will be stopped when has a working session for the headset is created successfully.
+        /// set isAutoRefreshHeadset
+        /// isAutoRefreshHeadset =  true, it will re-scan automatically when get warningCode = 142, until there is a headset connected.
+        /// </summary>
+        public void SetAutoRefreshHeadset(bool isAutoRefreshHeadset) {
```

Comment:
be hard to say. but IMO we should not support this function. sorry a Tung. we only need RefreshHeadset() function in Unity Plugin.  i meant, we don't suggest and support it.